examples/12_streamlit_demo.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2021 NREL
```

Comment:
Should this be renamed to example 11?

Reply:
or we can save 11 for a mulit turbine example?